mosaic/mosaic.go

```diff
@@ -134,7 +134,7 @@ func (m Mosaic) Dither(dither bool) Mosaic {
 	return m
 }
 
-// Set ThresholdLevel on Mosaic
+// Set ThresholdLevel on [Mosaic].
 func (m Mosaic) Threshold(threshold uint8) Mosaic {
```

Comment:
Let's use an `int` here
```suggestion
func (m Mosaic) Threshold(threshold int) Mosaic {
```

Reply:
talked to @aymanbagabas privately, we receive int and convert to uint8 internally